content/en/database_monitoring/setup_postgres/advanced_configuration.md

```diff
@@ -39,7 +39,8 @@ Add the `quantize_sql_tables` option to your database instance configuration in
 instances:
   - dbm: true
     ...
-    quantize_sql_tables: true
+    obfuscator_options:
+      quantize_sql_tables: true
```

Comment:
👋  this option is actually being phased out. Can you replace with:

```suggestion
      replace_digits: true
```

Otherwise LGTM!

Reply:
Pushed another commit to use replace_digits.